src/js/themes/hpe.js

```diff
@@ -106,31 +106,26 @@ export const hpe = deepFreeze({
       face: `
         @font-face {
           font-family: "Metric";
-          src: url("https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff2") format('woff2'),
-               url("https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff") format('woff');
+          src: url("https://hpefonts.s3.amazonaws.com/web/MetricHPE-Web-Regular.woff") format('woff');
```

Comment:
Hmmm.... I'm wondering if we can connect with someone who owns this S3 bucket? Since this is not on a CDN, folks accessing any of our apps from Ireland, Germany, Japan, India, Australia will be having to make intercontinental network requests.

hpe.com is making use of a CDN at this path: https://www.hpe.com/apps/hpeweb-ui/fonts/Metric-Regular.woff2, but notice it is not labelled MetricHPE.

Perhaps the Brand folks can help get us connected.

Reply:
Ooo thanks for catching that. I hadn't realized. I would agree we likely want to use the CDN you shared above